code/__HELPERS/_macros.dm

```diff
@@ -5,8 +5,30 @@
 // fun if you want to typecast humans/monkeys/etc without writing long path-filled lines.
 #define ishuman(A) istype(A, /mob/living/carbon/human)
 
+#define isjusthuman(A) A.type == /mob/living/carbon/human
+
 #define ismonkey(A) istype(A, /mob/living/carbon/monkey)
 
+#define isvox(A) istype(A, /mob/living/carbon/human/vox)
```

Comment:
Yeah remove these defines, this isn't how species work. While making a /mob/living/carbon/human/whatever WILL make a thing of that race, you can have something of that race that ISN'T /mob/living/carbon/human/whatever through setting species